.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

```suggestion
  - package-ecosystem: "github-actions" # See documentation for possible values
    directory: "/" # Location of package manifests
    groups:
      actions:
        patterns:
          - "*"
    schedule:
      interval: "weekly"
```

Let's group them to have a single PR for all actions. Docs here https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/optimizing-pr-creation-version-updates